src/services/school/hooks/index.js

```diff
@@ -226,6 +238,7 @@ exports.before = {
 		globalHooks.ifNotLocal(hasEditPermissions),
 		globalHooks.ifNotLocal(globalHooks.lookupSchool),
 		globalHooks.ifNotLocal(restrictToUserSchool),
+		validateSchoolNumber,
```

Comment:
should we not check also for create / update?

Reply:
There are no routes for create or update.
Only for .put/patch